### ISSUE TITLE
[Feature]: Geocoded location for person card

### ISSUE THREAD

**Issue by marcselman** [piitaya/lovelace-mushroom#825]
### Is your feature request related to a problem?

Currently the person card only shows Home or Away.
When someone is away it does not give a quick view of *where* someone is.

### Describe the solution you'd like

It would be great if we could select an entity that contains a geocoded location so we can show the actual address where someone is, if not at home.
In the Android app sensor settings you can enable this sensor so that an entity is automatically available in Home Assistant, for example: `sensor.pixel_6_pro_geocoded_location`.

**Comment by marcgeurts** [comment]
It does show where a person is when it's added as a zone in HA (not just Home and Away), so you can add zones like: Office, Shop etc.

iPhones also have the field `*_pro_geocoded_location`, so i could add an optional entity filter for that.

The problem is that a full size address doesn't really fit into the minimal design of the mushroom card.

What do you think @piitaya ?

**Comment by marcselman** [comment]
Yeah, I noticed that later as well. I guess you're right. It's a nice feature that you can see how far someone is when they're traveling home (instead of 'Away'), but you can always tap the tile to see the map of course.

But if you still would like to add the feature then letting the address scroll (like a ticker) would be very cool. I understand that's not that easy so a truncated address would suffice as well. Just seeing 'Grand Central Av...' for example is more helpful than 'Away' I think.

**Comment by piitaya** [comment]
For now, person entity only support zone, not_home and home state so Mushroom must only support this. You can use template if you want your own rules or mix entities.

**Comment by phillprice** [comment]
In case anyone stumbles across this - this is the template card I set up. 

One thing I would like to add is to get the badge working as in the person card, but I don't think `badge_icon` is template aware (also would need to get the current zone's icon somehow)

```
type: custom:mushroom-template-card
primary: >-
  {% if states('person.phill') == "home" %}

  Home

  {% elif states('person.phill') == "not_home" %}

  {% if state_attr('sensor.phillip_s_iphone_geocoded_location',"Areas Of
  Interest") != "N/A" %}

  {{ state_attr('sensor.phillip_s_iphone_geocoded_location','Areas Of Interest')
  }}

  {% else %}

  {{ state_attr('sensor.phillip_s_iphone_geocoded_location','Sub Locality') }}

  {% endif %} 

  {% else %}

  {{ states('person.phill')}}

  {% endif %}
secondary: |

  {{ state_attr('sensor.phillip_s_iphone_geocoded_location','Locality') }}
icon: mdi:home
picture: '{{state_attr(''person.phill'',''entity_picture'')}}'
```
